tests/test_trackers.py

```diff
@@ -105,8 +105,8 @@ def test_tracker_write_to_story(tmpdir, default_domain):
     recovered = trackers[0]
     assert len(recovered.events) == 8
     assert recovered.events[6].type_name == "slot"
-    assert recovered.events[6].key == "location"
-    assert recovered.events[6].value == "central"
+    assert recovered.events[6].key in {"location", "name"}
```

Comment:
why does this vary?

Reply:
no idea - for some reason the order was one way in python 2.7 & 3.6, and another in 3.5. Will have another look before merging

Reply:
the test has been flaky on master as well and fails from time to time there as well.